src/util/cover-model.html

```diff
@@ -32,6 +32,26 @@
     return this.stateObj.attributes.current_tilt_position === 0;
   });
 
+  addGetter('assumeState', function () {
+    return this.stateObj.attributes.assumed_state === true;
+  });
+
+  addGetter('disableButtonFullyOpen', function () {
```

Comment:
We should not add these to the cover model as they are very specific to the component where they are used. They should be computed inside the component instead.